.github/workflows/ci.yml

```diff
@@ -2,27 +2,39 @@
 name: CI
 
 on:
-  - push
   - pull_request
 
 permissions:
   contents: read
 
 jobs:
   rspec:
-    runs-on: ubuntu-20.04
+    name: >-
+      rspec (${{ matrix.ruby }})
```

Comment:
I don't like the name, but we need it to match with the required workflows.

Does anyone have an access to change them? Otherwise, we have to support outdated ones for ages 😕